src/build-change-graph.js

```diff
@@ -37,6 +37,11 @@ async function getPackageChangedFiles({
 
   let changedFiles = Array.from(new Set(committedChanges).union(dirtyChanges));
 
+  // if filename has space like `sample file.png` and its modified, that file will have double quotes in git output
+  // example: "some_directory/sample file.txt" and will endup with multiple double quotes when pushed to changedFiles array.
```

Comment:
Isn't the bug more like, if using path.join, you'll get `foo/"bar/sample file.png"`?

Reply:
> Isn't the bug more like, if using path.join, you'll get `foo/"bar/sample file.png"`?

yeah that will be case if path.join is used on this type of file